### ISSUE TITLE
Firefox extension references (but does not provide) font file 'fontawesome-webfont.woff2'

### ISSUE THREAD

**Issue by dholbert** [bitwarden/browser#575]
In Firefox, after using the bitwarden extension for a bit (e.g. clicking the browser toolbar icon), I see the following error appear in my Firefox Browser Console:

> downloadable font: download failed (font-family: "FontAwesome" style:normal weight:normal stretch:normal src index:1): status=2152857618 source: moz-extension://404b9dde-e4ee-4ef3-b64f-2ba1c38567bf/popup/fonts/fontawesome-webfont.woff2
>
>   unknown:7:11

If I visit the directory referenced in that URL ( moz-extension://404b9dde-e4ee-4ef3-b64f-2ba1c38567bf/popup/fonts/ ), I can see that there's a file called `fontawesome-webfont.woff` (note `.woff` in extension, not `.woff2`)

If I unzip the extension XPI file, I can find the relevant CSS in the file `popup/app.css`
````css
 * -------------------------- */
@font-face {
  font-family: 'FontAwesome';
  src: url(./fonts/fontawesome-webfont.eot);
  src: url(./fonts/fontawesome-webfont.eot?#iefix&v=4.7.0) format('embedded-opentype'),
       url(./fonts/fontawesome-webfont.woff2) format('woff2'),
       url(./fonts/fontawesome-webfont.woff) format('woff'),
       url(./fonts/fontawesome-webfont.ttf) format('truetype'),
       url(./fonts/fontawesome-webfont.svg#fontawesomeregular) format('svg');
````
...and I can confirm that the woff2 file referenced here does not exist in the extracted extension files.

Seems like either the woff2 file should be included in the extension, or else it shouldn't be referenced from the CSS.

**Comment by dholbert** [comment]
I have version 1.25.7 of the Firefox addon, BTW (installed today from https://addons.mozilla.org/en-US/firefox/addon/bitwarden-password-manager/ )

**Comment by kspearrin** [comment]
This is a known error that occurs because we only include the woff file to save space (it is the smallest one). woff2 is tried first so it throws the error and falls back to the woff file. We load the font awesome source directly from node_modules, so we can't really modify it. Since it doesn't hurt anything we haven't really tried fixing it any other way.

**Comment by kspearrin** [comment]
After looking at it again, it seems woff2 is actually the smallest file (76 vs 98 kb). I can't remember why I chose woff. Maybe woff has better compatibility across browsers?

**Comment by kspearrin** [comment]
Fixed in 1.26.0
